build/README.md

```diff
@@ -1,118 +1,106 @@
+- Google Cloud SDK (if deploying on GCP)
+  - Required if deploying to Google Cloud.
+- Golang.
+  - Required if developing the DSS codebase.
+- Optional - [Jsonnet](https://github.com/google/jsonnet)
+  - Recommended if editing the jsonnet templates.
 
 ## Docker images
 
-
-The grpc-backend and http-gateway are the 2 main binaries for processing DSS 
+The grpc-backend and http-gateway are the 2 main binaries for processing DSS
 requests. These are both built and pushed to a docker registry of your choice.
 You can easily find out how to push to a docker registry through a quick search.
 All major cloud providers have a docker registry service, or you can set up your
 own.
 
-
 ## Running locally
 
-
 Simply run the `run-locally.sh` script in this directory.
 
-
 ## Deploying the DSS on Kubernetes
 
-
 Note: All DSS instances in the same cluster must point their ntpd at the
 same NTP Servers. [CockroachDB recommends](https://www.cockroachlabs.com/docs/stable/recommended-production-settings.html#considerations)
 using [Google's Public NTP](https://developers.google.com/time/) when
 running in a multi-cloud environment.
 
-This secion discusses deploying a Kubernetes service, although you can deploy 
+This secion discusses deploying a Kubernetes service, although you can deploy
 a DSS instance however you like as long as it meets the CockroachDB requirements
 above. You can do this on any supported
 [cloud provider](https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/)
-or even on your own infrastructure.  Consult the Kubernetes documentation for
+or even on your own infrastructure. Consult the Kubernetes documentation for
 which ever provider you choose.
 
 1.  Create a new Kubernetes cluster as mentioned above. We recommend a new
     cluster for each DSS instance.
 
 1.  Create your static IP addresses. How you do this depends on your cloud
     provider:
-    *   1 for the HTTPS Gateway's Ingress. Note: if you're using Google Cloud
-        the HTTPS Gateway Ingress needs to be created as a "Global" IP address.
-    *   If you want to be able to join other clusters, you'll need static IP's 
-        for each of your CockroachDB nodes (min of 3).
-    *   [OPTIONAL] 1 for a loadbalancer across your CockroachDB nodes to provide
-        a single join target for other users.
+
+    - 1 for the HTTPS Gateway's Ingress. Note: if you're using Google Cloud
+      the HTTPS Gateway Ingress needs to be created as a "Global" IP address.
+    - If you want to be able to join other clusters, you'll need static IP's
+      for each of your CockroachDB nodes (min of 3).
+    - [OPTIONAL] 1 for a loadbalancer across your CockroachDB nodes to provide
+      a single join target for other users.
 
 1.  Use the `make-certs.py` script in this directory to create certificates for
     the new CockroachDB cluster:
 
         ./make-certs.py --cluster <cluster_context>
+            [--namespace <NAMESPACE>]
```

Comment:
Let's document how a user might choose a reasonable value for NAMESPACE

Reply:
See bulletpoint number 2 above, and let me know if you feel that is sufficient.

Reply:
Looks good above.  But brackets indicate optional parameters -- let's remove them for consistency with the new requirement in make-certs.py.